governance/polls/Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021.md

```diff
@@ -0,0 +1,86 @@
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_set being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP25
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The SAS-001 Core Unit aims to improve security, transparency and accessibility of the Maker Protocol by providing and maintaining auction services through open-source development.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP39/MIP39c2-Subproposals/MIP39c2-SP25.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP25)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp25-adding-sidestream-auction-services-core-unit-sas-001/10796)
+
+### MIP40c3-SP44
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP44 adds the MKR Incentive Plan budget for SAS-001: Sidestream Auction Services Core Unit. It contains:*
+*- Total MKR Expenditure Cap*
+*- Estimated MKR Expenditure (based on the current team)*
+*- Escrow Wallet mechanism*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP44.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP44)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp44-adding-sidestream-auction-services-core-unit-mkr-budget-sas-001/10802)
+
+### MIP41c4-SP26
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP41c4-SP26 onboards Daniel Kremerov(@danik) as Facilitator for the Sidestream Auction Services Core Unit.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP41/MIP41c4-Subproposals/MIP41c4-SP26.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP41c4SP26)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip41c4-sp26-sidestream-auction-services-facilitator-onboarding-sas-001/10797)
+
+## Outcomes
```

Comment:
Not so fast! I think you missed the [budget](https://forum.makerdao.com/t/mip40c3-sp43-adding-sidestream-auction-services-core-unit-budget-sas-001/10799) :P